packages/public-api/src/web.ts

```diff
@@ -23,30 +23,34 @@ export interface MaskNetworkAPIs {
     settings_setTrendingDataSource(provider: DataProvider): Promise<void>
     settings_getLaunchPageSettings(): Promise<LaunchPage>
 }
+
 export enum LaunchPage {
     facebook = 'facebook',
     twitter = 'twitter',
     dashboard = 'dashboard',
 }
+
 // This type MUST be sync with NetworkType in packages/web3-shared/src/types/index.ts
 export enum NetworkType {
     Ethereum = 'Ethereum',
     Binance = 'Binance',
     Polygon = 'Polygon',
     Arbitrum = 'Arbitrum',
 }
+
 export enum DataProvider {
     COIN_GECKO = 0,
-    COIN_MARKET_CAP = 1,
-    UNISWAP_INFO = 2,
+    COIN_MARKET_CAP,
```

Comment:
@guanbinrui IMO enums in this package MUST be explicitly marked about their value. This package is a public API and it's easy to make things wrong by inserting a new item in the middle of an enum.

Example:

```ts
enum Fruit {
    Apple = 0,
    Banana, // infered to be 1
}
```

And when I insert on item in the middle:

```ts
enum Fruit {
    Apple = 0,
    Pear, // 1
    Banana, // 2
}
```

This will make a breaking change in our public API too easy. We must avoid that.

Reply:
@Jack-Works enable https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-enum-initializers.md ?

Reply:
OK. Will revoke this change later.

Reply:
@septs is it possible to only open this rule in the `public-api` package?

Reply:
This rule might be useful across the entire repo.

Reply:
No, it's annoying when the enum is not part of public API (or stored somewhere).